python/pip.bzl

```diff
@@ -120,6 +98,14 @@ pip_import = repository_rule(
     implementation = _pip_import_impl,
 )
 
+def pip_import(**kwargs):
+    if "requirements_overrides" in kwargs:
+        # Overrides are serialized to string and passed to the rule, since
+        # rules cannot have deep dicts.
+        kwargs["requirements_overrides"] = str(kwargs["requirements_overrides"])
```

Comment:
Should we use a more standard format like json to ensure we don't have arbitrary types going around?

Reply:
Done.